### PR TITLE
[zh]Update content/zh/blog/_posts/2018-10-01-health-checking-grpc.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/blog/_posts/2018-10-01-health-checking-grpc.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ date: 2018-10-01
 --->
 **作者**： [Ahmet Alp Balkan](https://twitter.com/ahmetb) (Google)
 
+<!-- 
+**Update (December 2021):** _Kubernetes now has built-in gRPC health probes starting in v1.23.
+To learn more, see [Configure Liveness, Readiness and Startup Probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe).
+This article was originally written about an external tool to achieve the same task._
+-->
+**更新（2021年12月）:** _Kubernetes 从 v1.23 开始具有内置 gRPC 运行状况探测。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2018-10-01-health-checking-grpc.md:24]: 斜体在中文格式是不存在的，印刷体可以使用楷书。因此在本项目中英文斜体在翻译时，使用引号代替。
```suggestion
**更新（2021 年 12 月）：** “Kubernetes 从 v1.23 开始具有内置 gRPC 健康探测。
```

File: content/zh/blog/_posts/2018-10-01-health-checking-grpc.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ date: 2018-10-01
...
+了解更多信息，请参阅 [配置 Liveness、Readiness 和 Startup Probes](/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2018-10-01-health-checking-grpc.md:25]:
```suggestion
了解更多信息，请参阅[配置存活探针、就绪探针和启动探针](/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)。
```

File: content/zh/blog/_posts/2018-10-01-health-checking-grpc.md [hunk 3]
```diff
@@ -16,6 +16,15 @@ date: 2018-10-01
...
+本文最初是关于实现相同任务的外部工具。_
```
Comment (howieyuen) [inline content/zh/blog/_posts/2018-10-01-health-checking-grpc.md:26]:
```suggestion
本文最初是为有关实现相同任务的外部工具所写。”
```

**Review by howieyuen — CHANGES_REQUESTED** [review]
Thanks to the contribution, please refer to [中文本地化指南](https://kubernetes.io/zh/docs/contribute/localization_zh/) for more info.

**Comment by tengqm** [comment]
please squash your commits into one.

**Comment by kinzhi** [comment]
Please help to review this PR

**Pull request merged**